packages/components/src/components/hds/app-header/index.ts

```diff
@@ -42,6 +50,18 @@ export default class HdsAppHeaderComponent extends Component<HdsAppHeaderSignatu
     registerDestructor(this, (): void => {
       this.removeEventListeners();
     });
+
+    // TODO: Determine what defaults should be set:
+    // * Should hasA11yRefocus default to "true"? (I assume yes)
+    // * Should a11yRefocusSkipTo default to a value? (Could add an id to the SideNav then point to that id by default)
+
+    // if a11yRefocus is enabled, ensure that a11yRefocusSkipTo value is set
+    // if (this.hasA11yRefocus) {
+    //   assert(
+    //     '@a11yRefocusSkipTo for NavigatorNarrator (a11y-refocus) in "Hds::SideNav" must have a valid value',
+    //     this.args.a11yRefocusSkipTo !== undefined
+    //   );
+    // }
```

Comment:
I took this mainly from the `SideNav`.

However, the SideNav was checking `this.args.hasA11yRefocus` which is not true if it's not explicitly passed in (which it shouldn't have to be since the default is supposed to be `true`.)

I commented this out for now because I would need to pass an id into `a11yRefocusSkipTo` in all my examples otherwise they blow up as expected. But I wanted to confirm what behavior we actually want & want to enforce on consumers.

Reply:
I had thought that we would want to provide a skip link for both the AppHeader and the SideNav controls. If a user activated the AppHeader skip link I thought it would make sense to jump them to the SideNav where they could either skip again or else use the SideNav controls. @MelSumner how do you think this should work?

I think it could make sense to create an HDS `SkipLink` component in any case...

Reply:
I still don't know what to do about the SkipLinks.

I don't think the existing SideNav one works quite right, as I said, since it doesn't actually require consumers to add an id value for the Skip Link.

If we want to remove the built-in Skip Links and instead ask consumers to add them should I do that as part of this PR or is that something we should look at separately?

Reply:
**Q: How about I do the following?**

1. Create a reusable `HdsSkipLink` component.
2. Add this new SkipLink into the `AppHeader`.
3. Update the `SideNav` to replace its current built-in skip link with the new component.
4. When the SideNav is pared with the AppHeader, turn off its skip link and instead turn on the AppHeader skip link.

Reply:
I can't remember if we talked about this in sync, but are there times that the SideNav is used without the AppHeader?

Reply:
@shleewhite Currently the [SideNav](https://helios.hashicorp.design/components/side-nav) is always used without the AppHeader as the AppHeader is new and hasn't been adopted by products yet.

We plan to help teams in adopting the new AppHeader to pair it with the SideNav.

Reply:
> How about I do the following?

Makes sense to me

Reply:
We need to have the page navigation message there too, is there a reason we can't use `ember-a11y-refocus` like we've been using? That has a skip link (bypass block) built in.

Reply:
Ok, so what I've gone ahead and done for now, is to include a `NavigationNarrator` in the `AppHeader` component which is included by default. And, I set a default value of "#main" for `@skipTo` / `@a11yRefocusSkipTo`.

I _did not_ create a new SkipTo component as there doesn't seem to be alignment around that. (We can always do this later if desired.)